Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm

```diff
@@ -136,13 +140,77 @@ @interface UIViewController ()
     videoDimensionsChanged(model ? model->videoDimensions() : FloatSize());
 }
 
+void VideoPresentationInterfaceIOS::ensurePipPlacardIsShowing()
+{
+    if (m_pipPlacard) {
+        [m_pipPlacard setHidden:NO];
+        return;
+    }
+    auto pipPlacard = adoptNS([PAL::allocUIViewInstance() initWithFrame:[layerHostView() bounds]]);
+    pipPlacard.get().backgroundColor = (UIColor *)[PAL::getUIColorClass() blackColor];
```

Comment:
We generally use method syntax to avoid having `.get()` all over the code. Couple instances throughout the patch.

In this case you can `[pipPlacard setBackgroundColor:...]`.

Also is the cast really necessary?

Reply:
I think the cast is necessary - I was having compiling trouble without it, and clearUIColor() casts as well (I updated the patch with a grayUIColor() helper function to make the code a bit neater).